requirements.txt

```diff
@@ -32,3 +32,12 @@ snowballstemmer==1.2.0
 traitlets==4.0.0
 websocket-client==0.34.0
 wsgiref==0.1.2
+
```

Comment:
`git describe` (possibly with various formatting flags) will tell you the commit you're on, the most recently tagged ancestor, commit distance etc. Is that what you want?